mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensor.h

```diff
@@ -159,6 +159,13 @@ inline bool hasAnySparseOperandOrResult(Operation *op) {
   return hasAnySparseOperand(op) || hasAnySparseResult(op);
 }
 
+//
+// Inference.
+//
+
+AffineMap inferLvlToDim(AffineMap dimToLvl, MLIRContext *context);
```

Comment:
Add doc to both methods, for example

/// Given the dimToLvl map, infers the lvlToDim map, or returns empty Affine when inference fails.

and

/// Returns the lvlToDim map for the given dimToLvl map specific to the block sparse cases. Asserts on failure (so only use when known to succeed).

Or something like that